tests/test_mommy.py

```diff
@@ -243,12 +243,15 @@ def test_save_object_instances_when_handling_one_to_many_relations(self):
             owner=owner,
             _quantity=2,
         )
+
+        assert 0 == models.Dog.objects.count()  # ensure there're no dogs in our db
```

Comment:
I propose having the expected values on the right, like:
```
assert models.Dog.objects.count() == 0
```
Seems easier to read. What do you think? At some point, would be nice to agree on a code style too.
Also, the assertion lines are explicit so no need of comments for it.